src/components/note/NoteEditor.tsx

```
@@ -0,0 +1,144 @@
+  const handleChange = (value) => setValue(value);
+  const modules = useMemo(() => {
+    return {
+      toolbar: {
+        container: toolbarOptions,
+      },
+    };
+  }, []);
+
+  const {
+    control,
+    register,
```

ESLint

[error] 'register' is assigned a value but never used.

(no-unused-vars)